components/server/tsconfig.json

```diff
@@ -14,16 +14,15 @@
         "downlevelIteration": true,
         "module": "commonjs",
         "moduleResolution": "node",
-        "target": "es6",
+        "target": "ES2021",
```

Comment:
Searching for `"target": "` I find a bunch of other places where we emit `es6` still. Would it make sense to bump them all at once?